src/rpc/rawtransaction.cpp

```diff
@@ -208,10 +207,8 @@ static UniValue gettxoutproof(const JSONRPCRequest& request)
         throw std::runtime_error(
             RPCHelpMan{"gettxoutproof",
                 "\nReturns a hex-encoded proof that \"txid\" was included in a block.\n"
-                "\nNOTE: By default this function only works sometimes. This is when there is an\n"
-                "unspent output in the utxo for this transaction. To make it always work,\n"
-                "you need to maintain a transaction index, using the -txindex command line option or\n"
-                "specify the block in which the transaction is included manually (by blockhash).\n",
+                "You must either manually specify the blockhash of the block including the transaction\n"
+                "or maintain a transaction index, using the -txindex command line option."
```

Comment:
This doesn't currently compile, looks like you are missing a comma here.

Reply:
oops. fixed.